crates/wac-graph/tests/encoding.rs

```diff
@@ -331,6 +332,7 @@ fn encoding() -> Result<()> {
             })?
             .into_composition_graph(&path, test_case)
             .and_then(|graph| {
+                println!("{:?}", graph);
```

Comment:
Debugging `println!`?

Reply:
This has been useful for me. It should only show up if you pass `--nocapture` to cargo test. Happy to remove it if you find it distracting though.

Reply:
We can keep it in and remove it later should it prove too noisy.